test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -236,7 +256,39 @@ Verify the integration with the LDAP server for a regular user
 
    > You should be redirected to the 3scale main page.
 
-As the test is finished lets stop the EC2 instance.
+**Verify RHOAM uninstall**
```

Comment:
Is this to verify that on a cluster that's had rhoam uninstalled that if there is a reinstall later the logic for the cidr block still works as if it were a new one?

Reply:
We're not checking if CIDR block logic works correctly after reinstallation. At the moment, the test case for reinstallation is there to check that after reinstall, all required secrets (dms, smtp, pagerduty) are created (there was an issue with it before) and we're not waiting for installation to complete.

Do you think it would be worth to add the CIDR check to the reinstallation test case as well?

Reply:
No i think we should be fine there. Just thought this was connected to the cidr change as it was added in the same pr.